python/spear/log.py

```diff
@@ -5,9 +5,9 @@
 import inspect
 import os
 
-def log(message):
+def log(*message):
     current_frame = inspect.currentframe()
-    print(_log_get_prefix(current_frame) + message)
+    print(_log_get_prefix(current_frame) + ','.join([str(x) for x in message]))
```

Comment:
I don't think `,` should be here either. A couple of the files in this PR have been updated in the `ue-52` branch, including this one. Now that you're in a code-cleanup phase, it this is probably a good time to pull the latest changes from `ue-52`.

Reply:
Ah, yes, merged `ue-52` branch